packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -750,32 +750,3 @@ def generate_trade_nonce(
                 trade_nonce = random.randrange(0, MAX_UINT_256)  # nosec
             return {"trade_nonce": trade_nonce}
         raise NotImplementedError
-
-    @staticmethod
-    def get_last_code_id(ledger_api: LedgerApi) -> int:
```

Comment:
No point in these methods, just use API

Reply:
Sure, these methods are also unsafe, because when you deploy contract, someone else can also deploy contract before you get code_id.